modules/xfem/include/efa/EFAElement2D.h

```diff
@@ -127,6 +131,26 @@ class EFAElement2D : public EFAElement
 
   std::vector<EFANode *> getCommonNodes(const EFAElement2D * other_elem) const;
 
+  void addInteriorNode(EFAFaceNode * faceNode);
+  bool isInteriorNode(EFANode * node) const;
```

Comment:
Functions like this should be documented using the function-style doxygen comments with brief descriptions of what the functions do and what the arguments to them are. Describe the parameters with `@param` and the return (if applicable) with `@return`. Look at the declaration of `addGeomMarkedElem2D` in XFEM.h for an example.